config.mk

```diff
@@ -38,3 +38,10 @@ else
 	SHLIB_SUFFIX    :=.so
 endif
 endif
+
+SBOX=.cabal-sandbox
+ifneq "$(wildcard $(SBOX))" ""
+	IDRIS :=.cabal-sandbox/bin/idris
```

Comment:
The `$(IDRIS)` variable is only used to define the Idris version that will be used to: a) run the tests; b) clean the libraries; and (c) build/clean the documentation. This version will be the one that was last installed in the sandbox.

The libraries are still built as normal using the CABAL machinery.